android/app-example/src/androidTest/java/com/badoo/ribs/example/rib/hello_world/HelloWorldTest.kt

```diff
@@ -20,15 +21,15 @@ import org.junit.Test
 class HelloWorldTest {
 
     @get:Rule
-    val ribsRule = RibsRule { buildRib(it) }
+    val ribsRule = RibsRule { activity, savedInstanceState -> buildRib(activity, savedInstanceState) }
```

Comment:
Do we always use savedInstanceState? maybe we can "hide" optional dependencies in some context object that would be the context of the factory lambda (`RibsRule constructor(factory: Context.(activity) -> T)`)?